app/interfaces/api/v1/claim_params_helper.rb

```diff
@@ -2,6 +2,7 @@ module API
   module V1
     module ClaimParamsHelper
       extend Grape::API::Helpers
+      AGFS_SCOPE = %i[api v1 external_users claims advocate_claim params].freeze
```

Comment:
👍 

**[not a blocker]** 💭 Consider changing https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/app/interfaces/api/v1/external_users/claims/advocate_claim.rb#L7-L11 to make use of this new constant instead for DRYness.